### PR TITLE
docs: tweak container deployment info

### PR DESCRIPTION
**Pull request by daviddrysdale**

### CODE REVIEW

File: trillian/examples/deployment/kubernetes/README.md [hunk 1]
```diff
@@ -34,10 +33,13 @@ Process
...
    1. The `prefix:` to the URL path prefix where you want your log API to be served.
-   1. The `public_key:` and `private_key:` entries to your own keys.
+   1. The `public_key:` and `private_key:` entries to your
+      [own keys](../../../docs/ManualDeployment.md#key-generation).  (The
+      [`to_proto`](https://github.com/google/trillian-examples/gossip/testdata/to_proto)
+      utility can help with the conversion to protobuf format.)
 1. Run the [deploy.sh](deploy.sh) script, using the same `config.sh` file you
    used for your Trillian deployment:
-  `./deploy.sh ../../../../../trillian/examples/deployment/kubernetes/config.sh`
+  `./deploy.sh ${GOPATH}/src/github.com/google/trillian/examples/deployment/kubernetes/config.sh`
```
Comment (AlCutter) [inline trillian/examples/deployment/kubernetes/README.md]: apparently `GOPATH` can have multiple ':' separated elements... :/

File: trillian/examples/deployment/kubernetes/README.md [hunk 2]
```diff
@@ -20,12 +19,12 @@ Prerequisites
...
 
-Process
--------
+
+## Process
 
 1. Ensure that you've followed the instructions to [create a Trillian instance on
    GCP](https://github.com/google/trillian/tree/master/examples/deployment/kubernetes),
-   and have provisioned a suitable log tree into it.
+   and have provisioned a suitable log tree into it (and have the tree ID).
```
[the remark this answers is not in the document]
Comment (daviddrysdale) [reply trillian/examples/deployment/kubernetes/README.md]: Done

File: trillian/docs/ContainerDeployment.md [hunk 3]
```diff
@@ -1,3 +1,28 @@
...
+
+Further instructions describe how to deploy the core Trillian components on
+[Google Cloud Platform (GCP)](https://github.com/google/trillian/blob/master/examples/deployment/kubernetes/README.md),
+and on [Amazon Web Services (AWS)](https://github.com/google/trillian/blob/master/examples/deployment/aws/README.md).
+
+## CTFE Personality
+
+The `examples/deployment/` subdirectory includes
+[Kubernetes instructions](../examples/deployment/kubernetes/README.md)
+for deploying a CTFE personality on top of a deployment of the Trillian core
```
[the remark this answers is not in the document]
Comment (daviddrysdale) [reply trillian/docs/ContainerDeployment.md]: Done.

File: trillian/examples/deployment/kubernetes/README.md (hunk 1 above)
Comment (daviddrysdale) [reply trillian/examples/deployment/kubernetes/README.md]: Reverted to ugliness

**Comment by daviddrysdale** [comment]
PTAL

**Pull request merged**